test/units/module_utils/common/text/formatters/test_human_to_bytes.py

```diff
@@ -10,61 +10,60 @@
 from ansible.module_utils.common.text.formatters import human_to_bytes
 
 
+NUM_IN_METRIC = {
+    'K': 1024,
```

Comment:
I think it'd read better if you did
```suggestion
    'K': 2 ** 10,
```

Reply:
much better, thanks a lot!